module3/assignment3.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Camden Grove | INFO 2439 | Assignment3</title>
+  <meta charset="utf-8"> 
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+    <link rel="stylesheet" href="_css/Grove_a3.css">
+	<link href="https://fonts.googleapis.com/css?family=Notable&display=swap" rel="stylesheet">
+  <script src="https://code.jquery.com/jquery-1.11.1.min.js"></script>
+  <script src="https://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+    
+</head>
+<body>
+<!-- Camden Grove
+     Assignment3.html
+     INFO2439
+     Thoendel
+     9/29/19
+-->
+<div data-role="header">
+<a data-ajax="false" href="http://student.mccinfo.net/~cjgrove/info2439/index.html" class="ui-btn-left" target="_blank" data-icon="home">Home</a>
```

Comment:
You wouldn't have to include an absolute URL here. A relative - or better - dynamic back or home button - would be preferrable.